policyengine_us/variables/gov/states/ri/tax/income/agi/subtractions/social_security/ri_social_security_modification.py

```diff
@@ -0,0 +1,45 @@
+from policyengine_us.model_api import *
+
+
+class ri_social_security_modification(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Rhode Island Social Security Modification"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/Social%20Security%20Worksheet_w.pdf"
+    defined_for = "ri_social_security_modification_eligible"
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        head_or_spouse = person("is_tax_unit_head_or_spouse", period)
+        age = person("age", period)
+        birth_year = -(age - period.start.year)
```

Comment:
use variable

Reply:
Do we have a person level birth year variable? I can only see `older_spouse_birth_year` which is tax_unit

Reply:
If not I can create one here

Reply:
yes please create it and amend `older_spouse_birth_year` to derive from it